tests/accessor/accessor_api_common_buffer_local.h

```diff
@@ -231,7 +231,10 @@ struct buffer_accessor_get_pointer_r {
   template <typename accT, typename errorAccT>
   static void check(const accT& acc, const errorAccT& errorAccessor,
                     const sycl_id_t<dim>& offset) {
-    const auto expectedRead = expected_value_t::expected_read(offset);
+    sycl_id_t<dim>
+        zero_offset;  // In SYCL 2020, get_pointer() always returns the base
+                      // buffer pointer, ignoring any offset.
```

Comment:
To avoid this horrible formatting, we can move the comment from the line with the code to the line before the code.

E.g.
```c++
// get_pointer() always returns the base buffer pointer ...
sycl_id_t<dim> zero_offset;
```

Reply:
</3 clang-format.

Done. Thanks.